README.md

```diff
@@ -23,7 +23,8 @@ You’ll need to have Node 14.0.0 or later version on your local development mac
 To create a new app, you may choose one of the following methods:
 * `npx create-dm-app my-app`
 
-It will create a directory called my-app inside the current folder.
+It will create a directory called my-app inside the current folder. 
+Note: if you want to change the datasource name or add new data sources, the visibleDataSources list defined in [App.tsx](https://github.com/equinor/create-dm-app/blob/main/src/App.tsx) needs to be updated.
```

Comment:
Should not be in here. See comment below.

Reply:
fixed!